src/frontend/JasmineGraphFrontEnd.cpp

```diff
@@ -1065,6 +1065,50 @@ void *frontendservicesesion(std::string masterIP, int connFd, SQLiteDBInterface
             JasmineGraphServer *jasmineServer = new JasmineGraphServer();
             jasmineServer->inDegreeDistribution(graphID);
 
+            int result_wr_done = write(connFd, DONE.c_str(), FRONTEND_COMMAND_LENGTH);
+            if (result_wr_done < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+            result_wr_done = write(connFd, "\r\n", 2);
+            if (result_wr_done < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+        } else if (line.compare(DPCNTRL) == 0) {
+            frontend_logger.log("Duplicate Centralstore", "info");
+
+            int result_wr = write(connFd, SEND.c_str(), FRONTEND_COMMAND_LENGTH);
+            if (result_wr < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+            result_wr = write(connFd, "\r\n", 2);
+            if (result_wr < 0) {
+                frontend_logger.log("Error writing to socket", "error");
+                loop = true;
+                continue;
+            }
+
+            char graph_id[FRONTEND_DATA_LENGTH];
+            bzero(graph_id, FRONTEND_DATA_LENGTH + 1);
+            string name = "";
```

Comment:
Where do we use the variables `name` and `path`?

Reply:
removed

Reply:
Ok